### PR TITLE
logging and style updates

### PR DESCRIPTION
**Pull request by ginberg**
- add logging functionality and remove logging dependency
- add a style parameter for a custom dashboard color

### CODE REVIEW

File: R/generate_template.R [hunk 1]
```diff
@@ -215,6 +223,21 @@ create_new_application <- function(name, location, sampleapp = FALSE, resetbutto
...
+        ui_content  <- readLines(con = ui_file)
+        ui_content[length(ui_content)]     <- paste0(substr(ui_content[length(ui_content)], 1, nchar(ui_content[length(ui_content)]) - 1), ",")
+        white_space <- paste(rep(" ", ifelse(dashboard_plus, nchar("dashboardPagePlus"), nchar("dashboardPage"))), collapse = "")
+        ui_content[length(ui_content) + 1] <- sprintf("%s skin = '%s')", white_space, skin_value)
+        writeLines(ui_content, con = ui_file)
+        close(ui_file)
+    }
+    
+    
+
```
Comment (kartikeyakirar) [inline R/generate_template.R:240]: too many white lines

File: R/generate_template.R [hunk 2]
```diff
@@ -215,6 +223,21 @@ create_new_application <- function(name, location, sampleapp = FALSE, resetbutto
...
```
Comment (kar-agg-gen) [inline R/generate_template.R:241]: too many whitelines

File: R/generate_template.R [hunk 3]
```diff
@@ -105,11 +107,13 @@
 #' 
 #' # blank app named 'myblankapp' created in a temp dir
 #' create_new_application(name = 'myblankapp', location = tempdir())
+#' # blank app named 'myblankapp' with a green skin created in a temp dir
+#' create_new_application(name = 'myblankapp', location = tempdir(), style = (skin = "green"))
```
Comment (kar-agg-gen) [inline R/generate_template.R:111]: it should be style = list(skin = "green")

File: man/create_new_application.Rd [hunk 4]
```diff
@@ -129,10 +132,14 @@ rightsidebar = "table")
 
 # blank app named 'myblankapp' created in a temp dir
 create_new_application(name = 'myblankapp', location = tempdir())
+# blank app named 'myblankapp' with a green skin created in a temp dir
+create_new_application(name = 'myblankapp', location = tempdir(), style = (skin = "green"))
```
Comment (kar-agg-gen) [inline man/create_new_application.Rd:136]: it should be style = list(skin = "green")

File: R/logger.R [hunk 5]
```diff
@@ -0,0 +1,797 @@
+##
```
Comment (kar-agg-gen) [inline R/logger.R:1]: this file also has some extra white line but I did not address them as this function is directly coming from package function.

**Pull request merged**